src/tests/end-to-end/tests/target-page/visualization-boxes.test.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { Browser } from '../../common/browser';
+import { launchBrowser } from '../../common/browser-factory';
+import { PopupPage } from '../../common/page-controllers/popup-page';
+import { TargetPage } from '../../common/page-controllers/target-page';
+import { scanForAccessibilityIssues } from '../../common/scan-for-accessibility-issues';
+
+describe('Target Page visualization boxes', () => {
+    let browser: Browser;
+    let targetPage: TargetPage;
+    let popupPage: PopupPage;
+
+    beforeAll(async () => {
+        browser = await launchBrowser({ suppressFirstTimeDialog: true });
+        targetPage = await browser.newTargetPage();
+        popupPage = await browser.newPopupPage(targetPage);
+        await popupPage.gotoAdhocPanel();
+    });
+
+    beforeEach(async () => {
+        await popupPage.disableAllToggles();
+    });
+
+    afterAll(async () => {
+        if (browser) {
+            await browser.close();
+            browser = undefined;
+        }
+    });
+
+    const adhocTools = ['Automated checks', 'Headings', 'Landmarks'];
+
+    it.each(adhocTools)('should pass accessibility validation', async adhocTool => {
```

Comment:
When you use `.each`, it's good to include some sort of placeholder for the test parameter in the test name to make sure the tests end up with more actionable/unique names; here, if one of these cases failed, the test name would read as `should pass accessibility validation`, `should pass accessibility validation #1`, or `should pass accessibility validation #2`, respectively, but if you used a test name parameter like `'for ad-hoc tool "%s" should pass accessibility validation'`, you'd instead get test names for the individual cases like `for ad-hoc tool "Automated checks" should pass accessibility validation`.

https://jestjs.io/docs/en/api#testeachtable-name-fn-timeout has documentation about how the formatting works.